src/modules/universal-discovery/components/content-creator/content.creator.component.js

```diff
@@ -24,7 +24,19 @@ export default class ContentCreatorComponent extends Component {
 
     handlePublish() {
         this._refIframe.contentWindow.onbeforeunload = () => {};
-        this._refIframe.contentWindow.document.body.querySelector('#ezrepoforms_content_edit_publish').click();
+        this.submitForm();
+    }
+
+    submitForm() {
+        const contentEditSubmit = this._refIframe.contentWindow.document.body.querySelector('#ezrepoforms_content_edit_publish');
+        if(contentEditSubmit) {
+            contentEditSubmit.click();
```

Comment:
I think it's not a good approach. This will submit all existing forms. 
I think we should add some selector to the submit button in both forms like `data-action="submit"` or something like that and query for this selector.